sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -287,17 +314,20 @@ namespace Azure { namespace Core { namespace Http {
     // Response can contain no body, or either of next bodies (m_bodyBuffer or
     // bodyStream)
     std::vector<uint8_t> m_bodyBuffer;
-    Http::BodyStream* m_bodyStream;
+    BodyStream* m_bodyStream;
+    BodyType m_bodyType;
 
     Response(
         int16_t majorVersion,
```

Comment:
```suggestion
        uint16_t majorVersion,
```

Reply:
see https://github.com/Azure/azure-sdk-for-cpp/issues/142
Team agreed to not use unsigned. So, I will update to be an `int`

Reply:
okay, but you're also going to change other `uint16_t` to `int`, right?